test/inductor/test_torchinductor.py

```diff
@@ -976,6 +976,12 @@ def fn(a):
 
         self.common(fn, (torch.randint(4, (4,)),))
 
+    def test_dist(self):
+        def fn(a, b):
+            return torch.dist(a, b)
```

Comment:
It would be good to add some other cases as well, such as different `p` or different dtypes.